engine/install/debian.md

```diff
@@ -33,7 +33,7 @@ Older versions of Docker were called `docker`, `docker.io`, or `docker-engine`.
 If these are installed, uninstall them:
 
 ```console
-$ sudo apt-get remove docker docker-engine docker.io containerd runc
+$ for pkg in docker docker-engine docker.io containerd runc; do sudo apt-get remove -y $pkg; done
```

Comment:
I'm a bit concerned that the command becomes longer (and potentially more error-prone for users to follow); wondering if instead we should use `dpkg --remove` (as suggested in https://superuser.com/a/518871)

```suggestion
$ sudo dpkg --remove docker docker-engine docker.io containerd runc
```

Reply:
If we update, we should also update the ubuntu page with the same changes